### ISSUE TITLE
Android Controls 

### ISSUE THREAD

**Issue by vjp-israelmb** [alexbatalov/fallout2-ce#121]
Hi! I am really enjoying my time with the app, it's amazing! But if I could change something it would be the way you have to do two taps on the screen for left click and three for the right one.

I understand that if you make it so one tap equals one left click it could bring problems, like the app might not distinguish between moving the cursor and doing a left click and it might do both... But could it be possible to make it so one tap = left click and two taps = right click in the future without it giving problems?

Well, still, this is a great app and despite what my nitpick might suggest, I am loving every minute I spend playing the game on the app! Have a great day and thank you!

**Comment by nonocake** [comment]
Agreed, I played the DOS port of fallout 1 on my phone with the app "magic dosbox" and it had one tap for left click and two taps for right. not sure how they implemented it but it is possible.

**Comment by bouncingbenji** [comment]
Also om magic dosbox on android you have on-screen buttons right and left clicks...I'm just happy I can play fallout 1/2 on the go xx
